### PR TITLE
Application: use startup (), cleanup active window handling

### PR DESCRIPTION
**Pull request by danirabbit**
Actions should be created on `startup ()` not every time we activate. Use `present_with_time` for showing active window on activate

### CODE REVIEW

File: src/Application.vala [hunk 1]
```diff
@@ -37,42 +36,42 @@ public class Feedback.Application : Gtk.Application {
...
-        quit_action.activate.connect (() => {
-            if (main_window != null) {
-                main_window.destroy ();
+            if (settings.get_boolean ("window-maximized")) {
+                main_window.maximize ();
             }
-        });
+
+            main_window.show_all ();
+            add_window (main_window);
```
Comment (Marukesu) [inline src/Application.vala:71]: is this line necessary? Gtk.Application.add_window() is called internally when setting the aplication property of the Gtk.ApplicationWindow.

File: src/Application.vala [hunk 2]
```diff
@@ -37,42 +36,42 @@ public class Feedback.Application : Gtk.Application {
...
+        }
+
+        active_window.present_with_time (Gdk.CURRENT_TIME);
```
Comment (Marukesu) [inline src/Application.vala:74]: IIRC, Gtk.Window.present() has the same behavior of Gtk.Window.present_with_time(Gdk.CURRENT_TIME).

Comment (danirabbit) [reply src/Application.vala:74]: According the the docs, `present ()` [shouldn't be used](https://valadoc.org/gtk+-3.0/Gtk.Window.present.html) and it says to use `present_with_time ()` [when opening a window that's already open](https://valadoc.org/gtk+-3.0/Gtk.Window.present_with_time.html)

File: src/Application.vala (hunk 1 above)
Comment (danirabbit) [reply src/Application.vala]: Oh yup you're right. Fixed!

File: src/Application.vala (hunk 2 above)
Comment (Marukesu) [reply src/Application.vala:73]: Gtk.Window.present() just call Gtk.Window.present_with_time(Gdk.CURRENT_TIME).

Comment (danirabbit) [reply src/Application.vala]: Okay changed! Thanks for double checking it :)

**Review by Marukesu — APPROVED** [review]

**Pull request merged**